### PR TITLE
Refractor pixiu_start.go

### PR DESCRIPTION
**Pull request by lmhmhl**
Here are some changes as follow:
1. Refractor `pixiu_start.go`, which extract the inner method as an independent method and delete some useless blank line and comment.
2. Change the plugin registered by hand to auto using `init()` method when import the related package.

### CODE REVIEW

File: cmd/pixiu/control.go [hunk 1]
```diff
@@ -18,6 +18,7 @@
 package main
 
 import (
+	"github.com/apache/dubbo-go-pixiu/pkg/model"
```
Comment (AlexStocks) [inline cmd/pixiu/control.go:21]: move it to the 3rd import block.

File: cmd/pixiu/control.go [hunk 2]
```diff
@@ -133,3 +97,47 @@ var (
...
+	logger.InitLog(logConfPath)
+	if err := GetAPIConfig(bootstrap, apiConfigPath); err != nil {
+		return err
+	}
+	InitNumCPU(c)
+	pixiu.Start(bootstrap)
+	return nil
+}
+
+func GetAPIConfig(bootstrap *model.Bootstrap, apiConfigPath string) error {
```
Comment (AlexStocks) [inline cmd/pixiu/control.go:119]: func GetAPIConfig(bootstrap *model.Bootstrap, apiConfigPath string) error {
	metaConfig := bootstrap.GetAPIMetaConfig()
	if metaConfig == nil {
		if _, err := config.LoadAPIConfigFromFile(apiConfigPath); err != nil {
			logger.Errorf("load api config error:%+v", err)
			return err
		}
                 return nil
         }

	if _, err := config.LoadAPIConfig(bootstrap.GetAPIMetaConfig()); err != nil {
		logger.Errorf("load api config from etcd error:%+v", err)
		return err
	}
	return nil
}

**Review by xiaoliu10 — APPROVED** [review]

**Review by williamfeng323 — APPROVED** [review]

**Comment by williamfeng323** [comment]
review dog的麻烦处理下

**Comment by mark4z** [comment]
this pr maybe can not merged, I will refractor the filter later. @lmhmhl @xiaoliu10

**Pull request closed without merging**